zero_bin/rpc/src/native/txn.rs

```
@@ -357,3 +398,228 @@ fn prestate_tracing_options(diff_mode: bool) -> GethDebugTracingOptions {
+        trace!("STEP: {:?}", step);
+        trace!("STEPS: {:?}", structlog_trace.struct_logs.len());
+        trace!("PREV OPCODE: {}", prev_op);
+        trace!("OPCODE: {}", entry.op.as_str());
+        trace!("CODE: {:?}", code_hash);
+        trace!("CTX:  {:?}", ctx);
+        trace!("EXCEPTION OCCURED: {:?}", exception_occurred);
+        trace!("PREV_DEPTH:  {:?}", prev_depth);
+        trace!("CURR_DEPTH:  {:?}", curr_depth);
+        trace!("{:#?}\n", entry);
+
+        match op {
+            "CALL" | "CALLCODE" | "DELEGATECALL" | "STATICCALL" => {
+                let callee_address = {
+                    // This is the same stack index (i.e. 2nd) for all four opcodes.  See https://ethervm.io/#F1
+                    let mut evm_stack = entry
+                        .stack
+                        .as_ref()
+                        .expect("No stack found in structLog.")
+                        .iter()
+                        .rev();
+                    let callee_raw = *evm_stack
+                        .nth(1)
+                        .expect("Stack must contain at least two values for a CALL instruction.");
+                    let lower_bytes = U160::from(callee_raw);
+                    Address::from(lower_bytes)
+                };
+
+                if precompiles().contains(&callee_address) {
+                    debug!("PRECOMPILE at address {} called.", &callee_address);
+                } else if callee_addr_to_code_hash.contains_key(&callee_address) {
+                    let code_hash = callee_addr_to_code_hash[&callee_address];
+                    call_stack.push((code_hash, next_ctx_available));
+                } else {
+                    // This case happens if calling an EOA. See also https://www.evm.codes/#00?fork=cancun
+                    debug!(
+                        "Callee address {} has no associated `code_hash`.",
+                        &callee_address
+                    );
+                }
```

I think it'd be clearer to just silently ignore this case, as we don't expect to perform anything on non-contract targets

---

I'd like to make it explicit, now that it burnt me once.  Downgraded to `trace!`.